src/R/Components/Impl/ConnectionManager/IConnection.cs

```diff
@@ -10,6 +10,8 @@ public interface IConnection: IConnectionInfo {
 
         Uri Uri { get; }
 
+        string ContainerName { get; }
```

Comment:
Does it need to be pesistent? May need to go to `IConnectionInfo`

Reply:
Dunno. We can always find out if it is a docker connection or not by inspecting docker ports.